src/waveform/waveformmarklabel.cpp

```diff
@@ -47,9 +47,9 @@ void WaveformMarkLabel::prerender(QPointF bottomLeft, QPixmap icon, QString text
 
     QPainter painter(&m_pixmap);
 
-    painter.setPen(backgroundColor);
+    painter.setPen(QColor(Qt::transparent));
```

Comment:
Why was this changed?

Reply:
To remove the distracting border: 

![grafik](https://user-images.githubusercontent.com/1777442/65082039-457a8780-d9a5-11e9-89dd-f4741a1ff128.png)

Reply:
okay